components/Inputs/TileSelectInput.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import placements from 'themes/placements';
+import sizes from 'themes/sizes';
+import colors from 'themes/colors';
+import fonts from 'themes/fonts';
+
+export const TileSelectInput = ({
+  iconSize, iconDefault, iconPressed, label, setValue, value,
```

Comment:
powinno sie dać od jakiegoś propsa uzależnić wielkość tego selecta. Pokombinuj tak żeby był props width, height i ustaw tak paddingi żeby wyglądało podobnie jak na Figmie, bo jeżeli bedzie gdzieś potrzebny jakiś mniejszy to sie go nie da zrobić. Do tego przekaż props style i wrzuć do View które bedzie otaczać FImage (trzeba dodać to View tam). Ogólnie te wszystkie touchable nie działaja jak nie mają jako childrena view i dopiero w tym view coś innego